content/blog/2021-08-16-august-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,258 @@
+Amit continues with an explanation how data and model versioning works with
+Github paired with DVC.
+
+In a previous article entitled
+[Bring DevOps to Data Science with MLOps](https://www.analyticsvidhya.com/blog/2021/04/bring-devops-to-data-science-with-continuous-mlops/)
+Amit walks through a tutorial using CML to bring CI/CD functionality to your ML
+project and automate the process. All great posts to check out!👇🏼
+
+<external-link
+href="https://www.analyticsvidhya.com/blog/2021/06/mlops-tracking-ml-experiments-with-data-version-control/?utm_source=dlvr.it&utm_medium=twitter"
+title="Tracking ML Experiments With Data Version Control"
+description="Amit Kulkarni's tutorial on getting started with DVC and tracking eperiments"
+link="https://analyticsvidhya.com"
+image="/uploads/images/2021-08-16/a-v.png"/> <external-link
+href="https://www.analyticsvidhya.com/blog/2021/06/mlops-versioning-datasets-with-git-dvc/"
+title="MLOps | Versioning Datasets with Git & DVC"
+description="Amit Kulkarni's tutorial on how to DVC works with Git to version your datasets."
+link="https://analyticsvidhya.com"
+image="/uploads/images/2021-08-16/a-v.png"/> <external-link
+href="https://www.analyticsvidhya.com/blog/2021/04/bring-devops-to-data-science-with-continuous-mlops/"
+title="Bring DevOps To Data Science With MLOps"
+description="Amit Kulkarni's tutorial on how to use CML to bring the CI/CD functionality of DevOps to your data science projects."
+link="https://analyticsvidhya.com"
+image="/uploads/images/2021-08-16/a-v.png"/>
+
+## Andreas Malekos' MLOps Tool Stack at Continuum Industries
+
+Last but not least, we bring you a great article from
+[Andreas Malekos](https://www.linkedin.com/in/andreasmalekos/), Chief Scientist
+at [Continuum Industries](https://www.continuum.industries/). In
+[the post](https://neptune.ai/blog/mlops-tool-stack-continuum-industries) he
+outlines the tool stack and MLOPs platform they use to do their work automating
+and optimizing the design of linear infrastructure assets like water pipelines,
+overhead transmission lines, subsea power lines, or telecommunication cables.
+
+Amongst their tool stack are DVC and CML, and the article outlines what they
+like (!🙈Spoiler alert🙊! DVC making repeatability achievable) and the things
+that they don't like that still need to be improved.
+
+![Continuum Industries MLOps Tool Stack](/uploads/images/2021-08-16/continuum-tool-stack.png)
+_Continuum Industries MLOps Tool Stack
+[Source link](https://neptune.ai/wp-content/uploads/Continuum-Industries-tool-stack-final.png)_
+
+# DVC News
+
+Though the team has been taking some vacation time in the last month, there's
+still a lot going on!
+
+![Typing Cat](https://media.giphy.com/media/aNqEFrYVnsS52/giphy.gif)
+
+## Docs Updates
+
+This month we are introducing docs updates! We will be including this in each
+month's Heartbeat so that you will always be aware of the what has changed as
```

Comment:
Can you remind us the rationale behind sharing docs updates via blog? I'm not opposed but I'd like to discuss the scope with the team. The current list below is more detailed than what we even share internally with the company 🙂

Reply:
I originally suggested that we should communicate somehow the docs improvements being added. However I don't have a strong opinion on where (hearbeat, blog post, other) or how to aggregate the updates.

Although I agree partially with:

> all content should organically reach readers via search engines, support channels, etc

I think that docs should showcase/promote their updates similar to how releases work for software (i.e. [DVC](https://github.com/iterative/dvc/releases)/[CML](https://github.com/iterative/cml/releases)), where users can see a summary of the changes being added recently. 

I don't think it's realistic to expect that an user looking for info will be periodically re-searching the docs for updates or re-asking in support channels. In my experience, when you have a need, you look for info at one point in time and just deal with what it is published at that moment.

A real example:  

Last week I was looking for info regarding sharing experiments. I found a [section in Getting Started](https://dvc.org/doc/start/experiments#sharing-experiments) and then just the command reference. I used that succinct info and tried things by myself. My life would have been easier if I had access to the [Sharing Experiments User Guide](https://dvc.org/doc/start/experiments#sharing-experiments), which got merged a few days before my initial search.

I just notice that new guide because I was contributing to `dvc.org` and it happened to be an Open P.R. at the same time I checked the P.Rs. page. 

If I were an external user, how would I noticed this change that helped me?

Reply:
> I don't think it's realistic to expect that an user looking for info will be periodically re-searching the docs for updates or re-asking in support channels. In my experience, when you have a need, you look for info at one point in time and just deal with what it is published at that moment.

I agree with this.  And so it could be helpful to point out some of our changes.  My list may be more detailed - that's why I need help with what should be here.

I've heard now repeatedly from our Community that it seems people get things working for them and then may not check back. Even our champions that love us and use our tools: João - didn't know about Studio functionality, Sami and not using the experiments, I'm forgetting who, but someone using DVC, but yet to implement CML, though they want to.  All these people are busy making things right in their own teams.  To give them a monthly update on things that *may make their lives easier* is a very good idea.  (to the extent that they actually read Heartbeat - but adding this info may help drive more Heartbeat views)

Reply:
@jorgeorpinel I also think where Community members contribute significantly to docs changes we should shout that out too which is why a included @ meirale.  This will a.)recognize their contribution and b.) encourage more contribution.

Reply:
> I don't think it's realistic to expect that an user looking for info will be periodically re-searching the docs

@daavoo @jendefig Then it' also not realistic that they'll look in the blog for the same info.

Writing/reviewing this takes some effort so I think it's best to have a clear reason or problem we're solving here.

The question of how many website users are docs regulars and for what reasons they come back is interesting (I would assume mainly they return to the cmd ref.) but again it's not clear to me how we would serve them with these updates. Do they even overlap with blog readers? We just don't have all this info.

I think that in blog updates we should actual highlight product updates instead, and link to the new/improved docs that usually accompany those product releases. That (which we already do probably) feels more natural to me.

> to the extent that they actually read Heartbeat

Exactly. So maybe it may be also a matter of having a social-media friendly summary of heartbeats to post on Discord/Twitter along with a link to the full article. But that's up to @iterative/devrel I think.

Reply:
> where Community members contribute significantly to docs changes we should shout that out too which is why a included @ meirale

I like that idea but it could just be those docs updates we include in the heartbeat as part of the community section. BTW how will that person know he was mentioned?

Reply:
p.s. guys feel free to include in a [minute](https://www.notion.so/iterative/Minutes-35878778da26403ab78e08e783d2d58a) and join a docs meeting when you can if you'd like to discuss this with the whole team 🙂

Reply:
Quick recap on this: I believe we agreed to mark certain docs updates in our internal reports as "may be worth sharing with the community" and let #devrel take it from there. Thanks